core-utilities/src/main/java/edu/illinois/cs/cogcomp/core/utilities/Table.java

```diff
@@ -114,4 +184,20 @@ public String getFormattedString(double d, int numDecimalPlaces) {
     public void addSeparator() {
         separators.add(this.getRowCount());
     }
+
+    public static void main(String []args) {
```

Comment:
Can you remove the main method? 
If you want, you can move some of the logic under the test file as a unit test. (Though its fine to not have unit tests for table formatting).